java-frontend/src/test/java/org/sonar/java/se/ProgramPointTest.java

```diff
@@ -30,9 +30,9 @@
   public void test_to_string_method() throws Exception {
     // ToString method of program point is used by viewer.
     CFG cfg = CFGTest.buildCFG("void foo() {foo();}");
-    ProgramPoint pp = new ProgramPoint(cfg.blocks().get(0), 0);
+    ProgramPoint pp = new ProgramPoint(cfg.blocks().get(0));
     assertThat(pp.toString()).isEqualTo("B1.0  IDENTIFIER1");
-    pp = new ProgramPoint(cfg.blocks().get(0), 2);
+    pp = pp.next().next();
```

Comment:
Not really convinced by this notation. I would prefer to have the constructor still being visible, but annotated with `@VisibleForTesting`.